CMakeLists.txt

```diff
@@ -8,9 +8,22 @@ include(Options.cmake)
 
 # env
 set(LGRAPH_ROOT_DIR ${CMAKE_CURRENT_LIST_DIR})
-set(DEPS_INCLUDE_DIR ${CMAKE_CURRENT_LIST_DIR}/deps/install/include)
-set(DEPS_LIBRARY_DIR ${CMAKE_CURRENT_LIST_DIR}/deps/install/lib)
-set(DEPS_LIBRARY64_DIR ${CMAKE_CURRENT_LIST_DIR}/deps/install/lib64)
+# if DEPS_INCLUDE_DIR not set, use default
```

Comment:
fix the CI script under `tugraph-db/ci/`

Reply:
There is no need to fix since the previous build process  using default values also works.

Reply:
You can check PR passes the CI